clients/stellar-relay-lib/Cargo.toml

```diff
@@ -16,7 +16,7 @@ wallet = { path = "../wallet", features = ["testing-utils"] }
 
 [dependencies]
 hex = "0.4.3"
-log = {version = "0.4.14"}
+tracing = { version = "0.1", features = ["log"] }
```

Comment:
helps with [tokio-console and instrumentation](https://github.com/tokio-rs/console?tab=readme-ov-file#whats-all-this-then): 
> the [console-subscriber](https://github.com/tokio-rs/console/blob/main/console-subscriber) crate in this repository contains an implementation of the instrumentation-side API as a [tracing-subscriber](https://lib.rs/crates/tracing-subscriber) [Layer](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/layer/trait.Layer.html), for projects using [Tokio](https://tokio.rs/) and [tracing](https://lib.rs/crates/tracing).